posthog/queries/retention.py

```diff
@@ -280,14 +280,16 @@ def _retrieve_people_in_period(self, filter: RetentionFilter, team: Team):
 
     def process_people_in_period(
         self, filter: RetentionFilter, vals, people_dict: Dict[str, ReturnDict]
-    ) -> List[Dict[str, Any]]:
+    ) -> Dict[str, Any]:
         marker_length = filter.total_intervals
         result = []
+        appearance_totals = [0 for _ in range(marker_length)]
         for val in vals:
```

Comment:
I was having a lot of difficulty unpacking this code.  Nice way to make this more readable:

```
for person_id, _, appearances_array in vals:
```

Reply:
yup def better